packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -257,7 +258,8 @@ module.exports = {
     // having to parse `index.html`.
     new ManifestPlugin({
       fileName: 'asset-manifest.json'
-    })
+    }),
+    new BuildProgressPlugin()
```

Comment:
Can we add a small comment here explaining what it does. "Displays a progress bar during the build" is a fine way to put it.